src/hooks/useAfterMount.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2024 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DependencyList, EffectCallback, useEffect, useState } from 'react';
+
+export function useAfterMount(
```

Comment:
I'm just not sure about the hook name. If the dependencies are not changed after the first mount, the effect will never get called.

Reply:
You are right. 👍  Any suggestions to either rewrite the function or rename it?

Reply:
`useAfterMountOnly` ? Or something like `useEffectWithMount(runOnMount: boolean, runAfterMount: boolean)`, so you could also use this for mount-only effects, not sure if that makes sense :)